proposals/299-ip-failure-count.txt

```diff
@@ -136,7 +136,7 @@ Ticket: https://trac.torproject.org/projects/tor/ticket/27491
    The initial value of the failure points in the scenarios described in this
    section would be:
 
-      * IPv4 Faulure Points: Count the number of IPv6-capable relays
+      * IPv4 Failure Points: Count the number of IPv6-capable relays
 
       * IPv6 Failure Points: Count the number of IPv4-capable relays
 
```

Comment:
This comment is about the line below this line.

I think we should update the failure point values when we get a new consensus or bridge configuration, including the first consensus. We could turn the new information into affinity values between 0 and 4, and add them to the existing affinity values.

That way, we use the new information, but it doesn't overwhelm our history.

Reply:
Makes sense. I will do this.